src/util/helpers.tsx

```diff
@@ -290,3 +290,92 @@ export const getClientOS = (userAgent: string) => {
 
   return null;
 };
+
+export const getSupportedArchitectures = (
+  instanceType: "container" | "virtual-machine",
+): string[] => {
+  if (instanceType === "container") {
+    return [
+      "i686",
+      "x86_64",
+      "armv7l",
+      "aarch64",
+      "ppc",
+      "ppc64",
+      "ppc64le",
+      "s390x",
+      "mips",
+      "mips64",
+      "riscv32",
+      "riscv64",
+      "armv6l",
+      "armv8l",
+      "loongarch64",
+    ];
+  }
+
+  return ["x86_64", "aarch64", "ppc64le", "s390x"];
+};
```

Comment:
As we only are interested in the vm options, I think we can keep the four options as a constant in the ConvertToInstanceBtn.